sycl/source/detail/memory_manager.cpp

```diff
@@ -1286,6 +1286,122 @@ void MemoryManager::ext_oneapi_copy_cmd_buffer(
   }
 }
 
+void MemoryManager::ext_oneapi_copyD2H_cmd_buffer(
+    sycl::detail::ContextImplPtr Context, RT::PiExtCommandBuffer CommandBuffer,
+    SYCLMemObjI *SYCLMemObj, void *SrcMem, unsigned int DimSrc,
+    sycl::range<3> SrcSize, sycl::range<3> SrcAccessRange,
+    sycl::id<3> SrcOffset, unsigned int SrcElemSize, void *DstMem,
+    unsigned int DimDst, sycl::range<3> DstSize, sycl::range<3> DstAccessRange,
+    sycl::id<3> DstOffset, unsigned int DstElemSize,
+    std::vector<RT::PiExtSyncPoint> Deps, RT::PiExtSyncPoint *OutSyncPoint) {
+  (void)DstAccessRange;
```

Comment:
I think this is a remnant of the existing methods for copies that shared a common code path, but if we are calling separate functions from the Command then we should be able to just remove these unused parameters.